expfmt/openmetrics_create.go

```diff
@@ -169,6 +171,10 @@ func MetricFamilyToOpenMetrics(out io.Writer, in *dto.MetricFamily) (written int
 				metric.Counter.GetValue(), 0, false,
 				metric.Counter.Exemplar,
 			)
+			if metric.Counter.CreatedTimestamp != nil {
```

Comment:
Are we sure this will not add OpenMetrics confusing feature to all callers/importers of this functionality? It is used by client_golang which might decide to remove CT from dto, but it feels safer and generally easier to use if this function would accept another argument (or we create another function e.g. `MetricFamilyToOpenMetricsWithCTSeries`) to accomplish this in opt-in fashion?